SPECS/pg_jobmon.spec

```diff
@@ -0,0 +1,89 @@
+%global _version 1.3.1
+
+%define pg_dir /usr/pgsql-9.4
```

Comment:
Just like in IMCS, we need to remove the %define here, and mention the fact that --define must be passed on the command line when running rpmbuild.